### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by enolfc**
Update the usage info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,25 @@
 This is a simple probe that contacts a EGI Notebooks endpoint
 to get the latest status report of the service and return OK, CRITICAL or WARNING accordingly.
 
-## Usage
+## Usage
 
 ```
```
Comment (gwarf) [inline README.md]:
```suggestion
```sh
```

**Review by gwarf — CHANGES_REQUESTED** [review]
I proposed a very important fix, please check.

**Comment by enolfc** [comment]
> I proposed a very important fix, please check.

you needed to be in the AUTHORS :)

**Comment by gwarf** [comment]
> So where is the AUTHORS update? ;)

This is generated automatically by pbr, don't worry

**Review by gwarf — APPROVED** [review]

**Pull request merged**